content/hi/docs/reference/glossary/persistent-volume-claim.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Persistent Volume Claim
```

Comment:
Better to write PersistentVolumeClaim. The words “persistent”, ”volume“ and “claim” don't appear in Hindi.